### ISSUE TITLE
Feature request: Do not skip users verified through blocked organizations

### ISSUE THREAD

**Issue by burpen** [kheina-com/Blue-Blocker#129]
Or, an alternative idea: Add a way to block all affiliates of a given account.

**Comment by DanielleMiu** [comment]
not sure about what the ask is here, there's already an option to skip (or, not skip) users verified through an organization

**Comment by burpen** [comment]
What I'm suggesting is that if I have that option enabled and I encounter a user verified through an organization, if the organization they are verified through is blocked by me, then block that user rather than skipping them.

**Comment by rougetimelord** [comment]
I don't think that there's a 100% accurate way for this to be implemented without making an API call. As a stopgap we could check if the organization is in the internal block history, but if the block came from manually blocking it won't be caught.
